plugin.video.otaku/resources/lib/pages/nyaa.py

```diff
@@ -77,7 +77,7 @@ def _process_nyaa_episodes(self, url, episode, season=None):
                 ep_match = rex_ep.findall(title)
                 ep_match = list(map(int, list(filter(None, itertools.chain(*ep_match)))))
                 if ep_match and ep_match[0] != int(episode):
-                    regex_ep_range = r'\s\d+-\d+|\s\d+~\d+|\s\d+\s-\s\d+|\s\d+\s~\s\d+'
+                    regex_ep_range = r'\s(?:\d+(?:-~\d+)?)(?:-(?:\d+(?:-~\d+)?))?'
```

Comment:
`(?:-~\d+)` doesnt look right, you will not have `-` followed by `~`, you most probably will have one or the other.
So it may be better to `(?:[-~]\d+)`

Reply:
looks like you only changed in one place, it still has `https://www.reddit.com/r/DataHoarder/comments/1cjskad/subscenecom_final_dump_276_million_subtitles/` in the other place in the same line

Reply:
done man, thank you for looking over this. if anything else needs fixing let me know, or if things can be improved upon more id be glad to help when i can.